src/common/AdaptiveStream.cpp

```diff
@@ -696,26 +696,24 @@ bool AdaptiveStream::seek(uint64_t const pos)
   return false;
 }
 
-bool AdaptiveStream::getSize(unsigned long long& sz)
+bool AdaptiveStream::getSegmentBufferSize(size_t& size)
```

Comment:
I always find names like this strange. For me a getter should return what it gets. a function name like `segmentBufferSizeRetrieveable(size_t& size)` works better when used for flow control. But maybe it's just me.

Reply:
maybe i prefer better as `getCurrentSegmentBufferSize` since should get data from the actual segment
@glennguy opinion?

Reply:
I think what phunkyfish is saying is that having the name start with get makes it seem like it's following convention of naming getter/setter functions - implies that the function should return some value or object, not a success/fail. His suggestion goes along with naming the function based on return type but I also see your point that it's assigning a value to the input parameter by 'getting' it from the segment....

I'm not too fussed either way and won't complain if it goes in as-is and we refactor more later on. Perhaps an idea could be to return the size from the function instead of bool, and where the code would return false it now returns -1 and we act accordingly?

Reply:
ah ok i got it from the wrong angle...
no here we have unsigned value it is bad return negative value, then in this case i prefer rename it as
`retrieveCurrentSegmentBufferSize`

Reply:
Sorry, this comment was in no way supposed to hold up this PR. Just something for discussion 😉

Reply:
no problem this will make it easier for me to keep it in mind